posts/2019-09-27-Async-await-hits-beta.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "Async-await hits beta!"
+author: Niko Matsakis
+---
+
+Big news! As of this writing, **syntactic support for async-await is
+available in the Rust beta channel!** It will be available in the 1.39
+release, which is expected to be released on **November 7th, 2019**.
+Once async-await hits stable, that will mark the culmination of a
+**multi-year effort to enable efficient and ergonomic asynchronous I/O
+in Rust**. It will not, however, mark the end of the road: there is
+still more work to do, both in terms of polish (some of the error
+messages we get today are, um, [not great]) and in terms of feature
+set ([async fn in traits], anyone?).
+
+[not great]: https://github.com/rust-lang/rust/issues/64130
+[async fn in traits]: https://github.com/dtolnay/async-trait
+
+(If you're not familiar with what async-await is, don't despair!
+There's a primer and other details later on in this post!)
+
+### Async-await support in the ecosystem growing rapidly
+
+But async-await has never been the entire story. To make good use of
+async-await, you also need strong libraries and a vibrant ecosystem.
+**Fortunately, you've got a lot of good choices, and they keep getting
+better:** 
+
```

Comment:
Probably futures-rs should be mentioned in here, too.

Reply:
Oh, and `hyper` of course.

Reply:
@cramertj ok! Have there been any recent developments to cite in particular?

Reply:
For hyper, I see https://github.com/hyperium/hyper/issues/1805

Reply:
WRT futures? no, just pointing people at the futures-preview 0.3.0-alpha and giving a heads up that 0.3.0 will arrive alongside stable async/await.